### DISCUSSION TITLE
chromeos: window needs to be resized before it can be used

### DISCUSSION THREAD

**Discussion by alephnull** [kovidgoyal/kitty#6996] [Q&A] [answered]
When launched, the kitty window can be seen but typing into it does nothing. After resizing the window manually, it works just fine.
Also, the minimise, maximise/layout and close buttons are not shown on the window titlebar. 
I took a screen recording of the behaviour but it looks weird and so I didn't post it as I felt it would be just confusing.

**Comment by kovidgoyal** [comment, marked as answer]
That will be because chrome OS is using some broken Wayland compositor
or the other. If chromeos support Xwayland then set linux_display_server
to x11 in kitty.conf and you will be fine.

**Comment by alephnull** [comment]
Just a note that `linux_display_server auto` does not work.
